EventFilter/EcalRawToDigi/interface/MatacqRawEvent.h

```diff
@@ -7,69 +7,63 @@
 #include <cstddef>
 #include <vector>
 
-#if 0 //replace 1 by 0 to remove XDAQ dependency. In this case it is assumed the
-      //machine is little endian.
-#include "i2o/utils/endian.h" //from XDAQ
+#if 0                          //replace 1 by 0 to remove XDAQ dependency. In this case it is assumed the \
+                               //machine is little endian.
```

Comment:
I think that the two comment lines could be moved just before ```if 0``` and the problem with clang-format  leading to a warning will go away

Reply:
fixed